# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/praenti/cups/setprinter.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur ../cups-1.1.21-org/berkeley/Makefile ./berkeley/Makefile
--- ../cups-1.1.21-org/berkeley/Makefile	2004-09-26 16:42:06.254764248 +0200
+++ ./berkeley/Makefile	2004-09-26 16:43:33.658476864 +0200
@@ -57,11 +57,11 @@
 
 install:	all
 	$(INSTALL_DIR) $(BINDIR)
-	$(INSTALL_BIN) lpq $(BINDIR)
-	$(INSTALL_BIN) lpr $(BINDIR)
-	$(INSTALL_BIN) lprm $(BINDIR)
+	$(INSTALL_BIN) lpq $(BINDIR)/lpq_cups
+	$(INSTALL_BIN) lpr $(BINDIR)/lpr_cups
+	$(INSTALL_BIN) lprm $(BINDIR)/lprm_cups
 	$(INSTALL_DIR) $(SBINDIR)
-	$(INSTALL_BIN) lpc $(SBINDIR)
+	$(INSTALL_BIN) lpc $(SBINDIR)/lpc_cups
 
 
 #
diff -Nur ../cups-1.1.21-org/man/Makefile ./man/Makefile
--- ../cups-1.1.21-org/man/Makefile	2004-09-26 16:42:07.097636112 +0200
+++ ./man/Makefile	2004-09-26 16:49:54.448587968 +0200
@@ -70,25 +70,25 @@
 install:	all
 	$(INSTALL_DIR) $(MANDIR)/man1
 	for file in $(MAN1); do \
-		echo Installing $$file in $(MANDIR)/man1...; \
-		$(INSTALL_MAN) $$file $(MANDIR)/man1/`basename $$file man`1; \
+		inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(MANDIR)/man1/$$inst'_cups.1'; \
 	done
-	$(RM) $(MANDIR)/man1/cancel.1
-	$(LN) lp.1 $(MANDIR)/man1/cancel.1
+	$(RM) $(MANDIR)/man1/cancel_cups.1
+	$(LN) lp.1 $(MANDIR)/man1/cancel_cups.1
 	$(INSTALL_DIR) $(MANDIR)/man5
 	for file in $(MAN5); do \
-		echo Installing $$file in $(MANDIR)/man5...; \
-		$(INSTALL_MAN) $$file $(MANDIR)/man5/`basename $$file man`5; \
+		inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(MANDIR)/man5/$$(inst)'_cups.5'; \
 	done
 	$(INSTALL_DIR) $(AMANDIR)/man$(MAN8EXT)
 	for file in $(MAN8); do \
-		echo Installing $$file in $(AMANDIR)/man$(MAN8EXT)...; \
-		$(INSTALL_MAN) $$file $(AMANDIR)/man$(MAN8EXT)/`basename $$file man`$(MAN8EXT); \
+		inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(AMANDIR)/man$(MAN8EXT)/$$(inst)'_cups'.$(MAN8EXT); \
 	done
-	$(RM) $(AMANDIR)/man$(MAN8EXT)/reject.$(MAN8EXT)
-	$(LN) accept.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/reject.$(MAN8EXT)
-	$(RM) $(AMANDIR)/man$(MAN8EXT)/disable.$(MAN8EXT)
-	$(LN) enable.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/disable.$(MAN8EXT)
+	$(RM) $(AMANDIR)/man$(MAN8EXT)/reject_cups.$(MAN8EXT)
+	$(LN) accept_cups.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/reject_cups.$(MAN8EXT)
+	$(RM) $(AMANDIR)/man$(MAN8EXT)/disable_cups.$(MAN8EXT)
+	$(LN) enable_cups.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/disable_cups.$(MAN8EXT)
 	$(INSTALL_DIR) $(MANDIR)/cat1
 	for file in $(CAT1); do \
 		echo Installing $$file in $(MANDIR)/cat1...; \
diff -Nur ../cups-1.1.21-org/systemv/Makefile ./systemv/Makefile
--- ../cups-1.1.21-org/systemv/Makefile	2004-09-26 16:42:07.529570448 +0200
+++ ./systemv/Makefile	2004-09-26 16:52:35.884046048 +0200
@@ -60,26 +60,24 @@
 
 install:	all
 	$(INSTALL_DIR) $(SBINDIR)
-	$(INSTALL_BIN) accept $(SBINDIR)
-	$(RM) $(SBINDIR)/reject
-	$(LN) accept $(SBINDIR)/reject
+	$(INSTALL_BIN) accept $(SBINDIR)/accept_cups
+	$(LN) accept $(SBINDIR)/reject_cups
 	$(INSTALL_BIN) cupsaddsmb $(SBINDIR)
-	$(INSTALL_BIN) lpadmin $(SBINDIR)
-	$(INSTALL_BIN) lpinfo $(SBINDIR)
-	$(INSTALL_BIN) lpmove $(SBINDIR)
+	$(INSTALL_BIN) lpadmin $(SBINDIR)/lpadmin_cups
+	$(INSTALL_BIN) lpinfo $(SBINDIR)/lpinfo_cups
+	$(INSTALL_BIN) lpmove $(SBINDIR)/lpremove_cups
 	$(INSTALL_DIR) $(BINDIR)
-	$(INSTALL_BIN) cancel $(BINDIR)
+	$(INSTALL_BIN) cancel $(BINDIR)/cancel_cups
 	$(INSTALL_BIN) cupstestppd $(BINDIR)
-	$(RM) $(BINDIR)/disable
-	$(LN) ../sbin/accept $(BINDIR)/disable
-	$(RM) $(BINDIR)/enable
-	$(LN) ../sbin/accept $(BINDIR)/enable
-	$(INSTALL_BIN) lp $(BINDIR)
-	$(INSTALL_BIN) lpoptions $(BINDIR)
-	$(INSTALL_BIN) lpstat $(BINDIR)
-	-$(INSTALL_BIN) -m 4755 -o $(CUPS_USER) -g $(CUPS_GROUP) lppasswd $(BINDIR)
-	if test ! -x $(BINDIR)/lppasswd; then \
-		chmod 755 $(BINDIR)/lppasswd; \
+	$(LN) ../sbin/accept $(BINDIR)/disable_cups
+	$(LN) ../sbin/accept $(BINDIR)/enable_cups
+	$(INSTALL_BIN) lp $(BINDIR)/lp_cups
+	$(INSTALL_BIN) lpoptions $(BINDIR)/lpoptions_cups
+	$(INSTALL_BIN) lpstat $(BINDIR)/lpstat_cups
+	$(INSTALL_BIN) lppasswd $(BINDIR)/lppasswd_cups
+	-$(INSTALL_BIN) -m 4755 -o $(CUPS_USER) -g $(CUPS_GROUP) lppasswd $(BINDIR)/lppasswd_cups
+	if test ! -x $(BINDIR)/lppasswd_cups; then \
+		chmod 755 $(BINDIR)/lppasswd_cups; \
 	fi
 
 
